daru-io.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.15'
   spec.add_development_dependency 'dbd-sqlite3'
   spec.add_development_dependency 'dbi'
+  spec.add_development_dependency 'jsonpath'
```

Comment:
BTW, why only development? Or you're postponing the policy of "optional libraries requireing"?..